kotest-common/src/commonMain/kotlin/io/kotest/collections/seqs.kt

```diff
@@ -0,0 +1,20 @@
+package io.kotest.collections
+
+/**
+ * Returns an iterator that will repeat the values from this sequence.
+ * If the sequence is empty, then it will return an empty iterator.
+ */
+fun <A> Sequence<A>.loopedIterator(): Iterator<A> {
```

Comment:
This is no longer used, would we want to remove this for now?

Reply:
Yes good call.